### ISSUE TITLE
Feature request: generate tagged PDF

### ISSUE THREAD

**Issue by cut2run** [puppeteer/puppeteer#3298]
Tagged PDFs are required by certain accessibility aids. Puppeteer has access to full DOM and should be able to use this info to generate accessible PDF. Is this feature should be implemented inside the Puppeteer package?

I did some research and found the Chromium team has assigned task to [generate tagged PDFs](https://bugs.chromium.org/p/chromium/issues/detail?id=880113) . Is this feature will be available for Puppeteer users right away if Chromium team would implement it? Or even if the feature will be available from Chromium, puppeteer still has to expose it via some "option"?

Please clarify on availability of this request.

**Comment by aslushnikov** [comment]
>  Is this feature will be available for Puppeteer users right away if Chromium team would implement it? Or even if the feature will be available from Chromium, puppeteer still has to expose it via some "option"?

@cut2run It depends on how they'll implement it upstream. If all PDFs produced by Chrome will be tagged, then we won't need to do anything else.

On the other hand, if they'll add a "tagged pdf" checkbox to the "save as PDF.." UI, we'll have to plumb the feature through the DevTools protocol and then expose it in puppeteer as an option.

**Comment by cut2run** [comment]
Thanks for clarification, the following is the answer from the Chromium team ...

> This is just the first step, a lot more changes need to be made to Chrome (and more to Skia too) for this to work. When finished, though, I was assuming it would just be automatic - I don't see any reason why the user should need to do any extra steps to generate a tagged PDF, we should just always do the right thing.

Basically when work by Chromium team will be done, tagged PDF will be standard.

Reference: [https://bugs.chromium.org/p/skia/issues/detail?id=8148](https://bugs.chromium.org/p/skia/issues/detail?id=8148)

**Comment by corey-slade** [comment]
It looks like Chrome implemented this feature in Chrome 85 (see below), but I don't appear to have the ability to generate a tagged PDF yet unless I'm missing something? 

https://blog.chromium.org/2020/07/using-chrome-to-generate-more.html#:~:text=Using%20Chrome%20to%20generate%20more%20accessible%20PDFs,-Wednesday%2C%20July%2029&text=Starting%20in%20Chrome%2085%20(scheduled,%22Save%20as%20PDF%22%20destination.)
